textext/utility.py

```diff
@@ -259,6 +260,38 @@ def exec_command(cmd, ok_return_value=0):
     return out + err
 
 
+def check_minimal_required_version(actual_version_str, required_version_str):
```

Comment:
A better name for this function is `version_greater_than`. The "actual" and "minimal"/"required" nature of arguments passed by  ***a** caller* of the function is unrelated to the function definition.